ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__metal-4.15-nightly-x86.yaml

```diff
@@ -29,6 +29,7 @@ tests:
     cluster_profile: metal-perscale-cpt
     env:
       BAREMETAL: "true"
+      CRUCIBLE: "false"
```

Comment:
can we try false without ""?

Reply:
Will try to remove the `""` on the **_openshift-qe-installer-bm-deploy-commands.sh_** file instead

Reply:
We should be good to go now.. I fixed it in Jetlag.

Reply:
I think removing the `""` on the **_openshift-qe-installer-bm-deploy-commands.sh_** file had already worked

Reply:
https://github.com/openshift/release/blob/eb71fba8b4eb63ecb0c46abdcc59f6dc6f606d70/ci-operator/step-registry/openshift-qe/installer/bm/deploy/openshift-qe-installer-bm-deploy-commands.sh#L45